src/partials/header-content.hbs

```diff
@@ -19,34 +19,63 @@
       </div>
       <div class="navbar-end">
         <div class="navbar-item has-dropdown is-hoverable">
-          <div class="navbar-link">Processors</div>
+          <div class="navbar-link">AsciiDoc</div>
           <div class="navbar-dropdown">
-            <div class="navbar-item"><strong>Asciidoctor</strong></div>
-            <a class="navbar-item" href="https://github.com/asciidoctor/asciidoctor" target="_blank" rel="noopener">Source</a>
-            <a class="navbar-item" href="https://github.com/asciidoctor/asciidoctor/issues" target="_blank" rel="noopener">Issues</a>
+            <a class="navbar-item" href="{{relativize (resolvePageURL 'asciidoc::index.adoc')}}">Language</a>
+            <a class="navbar-item" href="{{relativize (resolvePageURL 'asciidoc::syntax-quick-reference.adoc')}}">Syntax Quick Reference</a>
+          </div>
+        </div>
+        <div class="navbar-item has-dropdown is-hoverable">
+          <div class="navbar-link">Processing</div>
+          <div class="navbar-dropdown">
+            <div class="navbar-item"><strong>Core Processors</strong></div>
+            <a class="navbar-item" href="{{relativize (resolvePageURL 'asciidoctor::index.adoc')}}">Asciidoctor <small>Ruby</small></a>
+            <a class="navbar-item" href="{{relativize (resolvePageURL 'asciidoctor.js::index.adoc')}}">Asciidoctor.js <small>JavaScript</small></a>
+            <a class="navbar-item" href="#">AsciidoctorJ <small>Java</small></a>
             <hr class="navbar-divider">
-            <div class="navbar-item"><strong>Asciidoctor.js</strong></div>
-            <a class="navbar-item" href="https://github.com/asciidoctor/asciidoctor.js" target="_blank" rel="noopener">Source</a>
-            <a class="navbar-item" href="https://github.com/asciidoctor/asciidoctor.js/issues" target="_blank" rel="noopener">Issues</a>
+            <div class="navbar-item"><strong>Extended Converters</strong></div>
+            {{!--
+            <a class="navbar-item" href="{{relativize (resolvePageURL 'asciidoctor-pdf::index.adoc')}}">Asciidoctor PDF</a>
+            <a class="navbar-item" href="{{relativize (resolvePageURL 'asciidoctor-epub3::index.adoc')}}">Asciidoctor EPUB3</a>
+            <a class="navbar-item" href="{{relativize (resolvePageURL 'asciidoctor-reveal.js::index.adoc')}}">Asciidoctor reveal.js</a>
+            --}}
+            <a class="navbar-item" href="#">Asciidoctor reveal.js</a>
+          </div>
+        </div>
+        <div class="navbar-item has-dropdown is-hoverable">
+          <div class="navbar-link">Tooling</div>
+          <div class="navbar-dropdown">
+            <div class="navbar-item"><strong>Build Plugins</strong></div>
+            <a class="navbar-item" href="{{relativize (resolvePageURL 'maven-plugin::index.adoc')}}">Maven Tools</a>
```

Comment:
I liked the language labels above, would it make sense to add them here too? That way anyone can get a quick view of options without have to know much about the tech.

Reply:
Could you please elaborate? As far as I know, Maven and Gradle can build more than one language.

Reply:
I'm fine with adding the label "Java" here if that's what you mean. While it's true they can build more than one language, they are still a Java tool.

Reply:
@mojavelinux got it! I am thinking for example the scenario of someone new going through the docs saying "This is cool, but how do I integrate it in me current code?".